crates/router/src/core/webhooks/webhook_events.rs

```diff
@@ -325,3 +338,33 @@ async fn get_account_and_key_store(
         }
     }
 }
+
+async fn validate_event_type(
+    event_class: &HashSet<EventClass>,
+    event_type: &mut HashSet<EventType>,
```

Comment:
Nit: Naming

```suggestion
    event_classes: &HashSet<EventClass>,
    event_types: &mut HashSet<EventType>,
```

Please make this change everywhere else you're accepting `HashSet<EventClass>` or `HashSet<EventType>` as function parameters.